streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/RoleServiceImpl.java

```diff
@@ -86,7 +86,7 @@ public void deleteRole(Long roleId) {
                 () ->
                     new IllegalArgumentException(String.format("Role id [%s] not found", roleId)));
     List<Long> userIdsByRoleId = memberService.findUserIdsByRoleId(roleId);
-    AssertUtils.isTrue(
```

Comment:
`Utils.required` will throw `IllegalArgumentException`, but old check isn't.

Too many checks are replaced to the `Utils.required` due to release, so it's better to add a TODO here. In the long run, it is unreasonable for all checks to throw `IllegalArgumentException`.

Reply:
> `Utils.required` will throw `IllegalArgumentException`, but old check isn't.
> 
> Too many checks are replaced to the `Utils.required` due to release, so it's better to add a TODO here. In the long run, it is unreasonable for all checks to throw `IllegalArgumentException`.

AssertUtils.isTrue is also throw `IllegalArgumentException`, about the exception, it can be improved in the future